src/crafting/CraftingManager.php

```diff
@@ -29,12 +29,15 @@
 use pocketmine\utils\BinaryStream;
 use pocketmine\utils\DestructorCallbackTrait;
 use pocketmine\utils\ObjectSet;
+use function count;
 use function spl_object_id;
 use function usort;
 
 class CraftingManager{
 	use DestructorCallbackTrait;
 
+	public const SMITHING_RECIPES_OFFSET = 200000;
```

Comment:
I think I mentioned before that I don't like this being here. It should be managed in the network layer.